docs/intro.md

```diff
@@ -12,12 +12,11 @@ Get started building applications with wasmCloud today and enjoy the freedom of
 
 wasmCloud is designed around the following core tenets:
 
-- Dead simple Distributed Applications
+- Dead simple distributed applications
 - Run anywhere
 - Secure by default
-- Save time, money, and effort
```

Comment:
@autodidaddict any big reservations on removing this one? I don't have an issue with focusing on the other traits